src/main.rs

```diff
@@ -272,11 +277,13 @@ impl RpcService {
                         println!("work_generate completed in {}ms for difficulty {:#x}",
                             start.to(end).num_milliseconds(),
                             difficulty);
-                        let work: Vec<u8> = work.iter().rev().cloned().collect();
```

Comment:
@PlasmaPower it seems to work without this line, what do you think?

Reply:
That line reverses the order of the bytes. Are you sure the work it outputs is still valid without that?